docker-run.sh

```diff
@@ -54,13 +55,29 @@ EOF
 \$dolibarr_main_document_root_alt='/var/www/html/custom';
 \$dolibarr_main_data_root='/var/www/documents';
 \$dolibarr_main_db_host='${DOLI_DB_HOST}';
-\$dolibarr_main_db_port='${DOLI_DB_HOST_PORT}';
+\$dolibarr_main_db_port='3306';
 \$dolibarr_main_db_name='${DOLI_DB_NAME}';
 \$dolibarr_main_db_prefix='llx_';
 \$dolibarr_main_db_user='${DOLI_DB_USER}';
 \$dolibarr_main_db_pass='${DOLI_DB_PASSWORD}';
 \$dolibarr_main_db_type='mysqli';
+\$dolibarr_main_authentication='$auth';
```

Comment:
As PHP is using the $ to spec var, I prefer to differenciate with bash replaced value by upper case var name and encapsulated `${MY_VAR}` style.

```suggestion
\$dolibarr_main_authentication='${AUTH}';
```